examples/Makefile

```diff
@@ -27,19 +27,31 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
-
 EXAMPLES := adder/tests \
-	    endian_swapper/tests \
             axi_lite_slave/tests \
+            dff/tests \
+            endian_swapper/tests \
+            mean/tests \
+            mixed_language/tests \
+            ping_tun_tap/tests
 
 .PHONY: $(EXAMPLES)
 
 .PHONY: all
 all: $(EXAMPLES)
 
 $(EXAMPLES):
+ifeq ($(TRAVIS),true)
+	@echo "travis_fold:start:$@"
+endif
 	@cd $@ && $(MAKE)
+ifeq ($(TRAVIS),true)
+	@echo "travis_fold:end:$@"
+endif
```

Comment:
Please don't add these markers in our general Makefile; we don't want to expose our CI specialties there. (The markers don't add much value anyways from my PoV.)

Reply:
Ok, they are not part of the PRs any longer.